homeassistant/components/profiler/manifest.json

```diff
@@ -3,6 +3,7 @@
   "name": "Profiler",
   "codeowners": ["@bdraco"],
   "config_flow": true,
+  "disabled": "Dependencies does not support Python 3.12",
```

Comment:
This is only guppy3 that is problematic?

In that case, we should follow this one: <https://github.com/home-assistant/core/pull/88136>